prisma/schema.prisma

```diff
@@ -156,13 +158,17 @@ model TeacherReg {
 }
 
 model User {
-  id             Int       @id @default(autoincrement())
+  id             Int           @id @default(autoincrement())
   name           String?
-  email          String?   @unique
-  email_verified DateTime? @db.Timestamptz(6)
+  email          String?       @unique
+  email_verified DateTime?     @db.Timestamptz(6)
   image          String?
-  created_at     DateTime  @default(now()) @db.Timestamptz(6)
-  updated_at     DateTime  @default(now()) @db.Timestamptz(6)
+  created_at     DateTime      @default(now()) @db.Timestamptz(6)
+  updated_at     DateTime      @default(now()) @db.Timestamptz(6)
+  parents        Parent?       @relation("parentsTousers")
+  program_admins ProgramAdmin? @relation("program_adminsTousers")
+  teachers       Teacher?      @relation("teachersTousers")
+  volunteers     Volunteer?    @relation("usersTovolunteers")
```

Comment:
can the relation be named `usersToVolunteers` to match proper camel casing - same comment for other relations

Reply:
I don't think we ever will use `usersToVolunteers` in code so I don't think it's too much of a priority. One thing I do think we could change is to change the field from plural to singular. Thoughts?

Reply:
is it a 1:1 relationship?

Reply:
The change will persist and will not be reverted the next time we introspect. See [here](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names).

![Uploading image.png…]()

Reply:
Similar to how we have `@@mapping` for the model, we can use `@mapping` for fields!

![Uploading image.png…]()

Reply:
We can do this as another PR

Reply:
sounds good to me!